README.md

```diff
@@ -1,2 +1,118 @@
-# juttle-engine
-juttle execution engine
+# Juttle Engine
+
+[![Build Status](https://travis-ci.org/juttle/juttle-engine.svg?branch=master)](https://travis-ci.org/juttle/juttle-engine)
+
+Juttle Engine is an integrated environment for developing and executing
+[juttle](https://github.com/juttle/juttle) programs and visualizations. It lets
+you run Juttle programs stored on the file system and present the results in a
+browser for experimentation, development, deployment, and debugging of juttle
+programs.
+
+When using Juttle to power data visualizations, they cannot be rendered in the
```

Comment:
This reads too referential to the CLI, there's another ref on line 19. If people are likely to come to this repo as "main one" after briefly looking at juttle the language, there's no need to compare to what CLI can/cannot do.